### PR TITLE
Updating tracer

### PR DESCRIPTION
**Pull request by Filter94**
This PR implements issue(s) #

### Checklist

* [ ] I wrote new tests for my new core changes.
* [ ] I have successfully ran tests, style checker and build against my new changes locally.
* [ ] I have informed the team of any breaking changes if there are any.

### CODE REVIEW

File: .github/workflows/postman-testing.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   run-tests:
-    runs-on: [self-hosted, ubuntu-22.04, X64, medium]
```
Comment (bradbown) [inline .github/workflows/postman-testing.yml:17]: These shouldn't work, they large runners are missing the `ubuntu-22.04` still so need `Linux` for now temporarily, same for the other large runners below.

Comment (Filter94) [reply .github/workflows/postman-testing.yml:17]: Do we know the exact version of linux they have available?

Comment (bradbown) [reply .github/workflows/postman-testing.yml:17]: It's ubuntu-22.04 but the labels are based on what's [here](https://github.com/Consensys/linea-monorepo/actions/runners?tab=self-hosted).

**Review by bradbown — APPROVED** [review]

File: .github/workflows/postman-testing.yml (hunk 1 above)
Comment (Filter94) [reply .github/workflows/postman-testing.yml:17]: I reverted this change

**Review by diana-borbe-consensys — APPROVED** [review]

**Pull request merged**